Makefile

```diff
@@ -26,7 +35,7 @@ install:
 	cp runc /usr/local/bin/runc
 
 clean:
-	rm runc
+	rm runc || true
```

Comment:
You can get unfailable removal with `rm -f runc`.  No need for the `true` fallback.